src/lib/connectors/repos/queries/licence-version-purpose-conditions.js

```diff
@@ -0,0 +1,17 @@
+exports.findLicenceVersionPurposeConditionsByLicenceId = `
+  SELECT lvpc.licence_version_purpose_condition_id as "licenceVersionPurposeConditionId", lvpc.param_1 as param1, lvpc.param_2 as param2, lvpc.notes
+  FROM water.licence_version_purpose_conditions lvpc 
+  JOIN water.licence_version_purposes lvp on lvp.licence_version_purpose_id = lvpc.licence_version_purpose_id
+  JOIN water.licence_versions lv on lv.licence_version_id = lvp.licence_version_id
+  JOIN water.licences l on lv.licence_id = l.licence_id
```

Comment:
In both these queries the final join to `water.licences` may not be necessary as the `licence_id` value is present in `water.licence_versions`

Reply:
Indeed. Good point. Will remove the join. :)